tests/conftest.py

```diff
@@ -1871,6 +1872,26 @@ def upgrade_namespace_scope_session(admin_client, unprivileged_client):
     )
 
 
+@pytest.fixture(scope="session")
+def upgrade_namespaces(upgrade_namespace_scope_session, kmp_enabled_namespace):
+    return [kmp_enabled_namespace, upgrade_namespace_scope_session]
```

Comment:
Any reason to keep this fixture? only used by migratable_vms and the namesapces can be directly defined there

Reply:
ack